tests/unit/fem/test_build_dof_to_arrays.cpp

```diff
@@ -210,30 +247,59 @@ TEST_CASE("Build Dof To Arrays (Parallel)",
                fec = new L2_FECollection(order-1, dim);
             }
             ParFiniteElementSpace fespace(&pmesh, fec);
-            HYPRE_Int size = fespace.GetTrueVSize();
-
-            fespace.BuildDofToArrays();
+            int size = fespace.GetNDofs();
 
+            // Check elements
             for (int i = 0; i<size; i++)
             {
                int e = fespace.GetElementForDof(i);
                int l = fespace.GetLocalDofForDof(i);
 
-               if (e < 0 || e >= pmesh.GetNE()) { num_elem_fails++; }
+               if (e < 0 || e >= pmesh.GetNE()) { num_elem_fails++; continue; }
 
                fespace.GetElementDofs(e, dofs);
 
                if (l < 0 || l >= dofs.Size()) { num_rang_fails++; }
 
-               int ldof = (dofs[l] >= 0) ? dofs[l] : (-1 - dofs[l]);
+               int elemdof = (dofs[l] >= 0) ? dofs[l] : (-1 - dofs[l]);
+
+               if (i != elemdof) { num_elemdof_fails++; }
+            }
+
+            // Check bdr elements
+
+            // Get all boundary ldofs
+            Array<int> bdr(1); bdr = 1;
+            static_cast<FiniteElementSpace>(fespace).GetEssentialVDofs(bdr,
```

Comment:
```suggestion
            static_cast<FiniteElementSpace&>(fespace).GetEssentialVDofs(bdr,
```
You forgot reference here, it calls constructor then 😮

Reply:
Thank you!